frontend/src/coin/components/CoinBalanceItem.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
why this?

Reply:
We are not using ```  chainId,``` for now, but I don't want to remove it yet